Content.Server/Botany/Systems/BotanySwabSystem.cs

```diff
@@ -30,9 +30,9 @@ private void OnExamined(EntityUid uid, BotanySwabComponent swab, ExaminedEvent a
         if (args.IsInDetailsRange)
         {
             if (swab.SeedData != null)
-                args.PushMarkup(Loc.GetString("swab-used"));
+                args.PushMarkup(Loc.GetString("botany-swab-used"));
```

Comment:
Why not use the existing FTL strings?

Reply:
New item new FTL string made sense in my head. Else I would have just changed the describtion and not make a new item?
Not much of a medical player so I have no idea if they are mapped outside of botany, which could lead to confusion, was my thinking.